Foundation.UnitTests build broken on OSX

In file included from ../../frameworks/OSXShims/src/LoggingNative.mm:17:
./../../../include/xplat/LoggingNative.h:38:1: error: unknown type name '__declspec'
LOGGING_EXPORT void TraceVerbose(const wchar_t* tag, const wchar_t* format, ...);
^
./../../../include/xplat/LoggingNative.h:24:35: note: expanded from macro 'LOGGING_EXPORT'
#define LOGGING_EXPORT extern "C" LOGGING_IMPEXP
                                  ^
./../../../include/xplat/LoggingNative.h:19:24: note: expanded from macro 'LOGGING_IMPEXP'
#define LOGGING_IMPEXP __declspec(dllimport)
                       ^
./../../../include/xplat/LoggingNative.h:38:15: error: expected ';' after top level declarator
LOGGING_EXPORT void TraceVerbose(const wchar_t* tag, const wchar_t* format, ...);
              ^
              ;

...

14 errors generated.
make[1]: *** [../../../build/Tests/UnitTests/Foundation/OSX/Objects/LoggingNative.o] Error 1
make: *** [Foundation.UnitTests.exe] Error 2

--- a/tests/unittests/Foundation/Makefile
+++ b/tests/unittests/Foundation/Makefile
@@ -32,6 +32,7 @@ CFLAGS += -x objective-c++
 CFLAGS += $(INC_PARAMS)
 CFLAGS += -DTARGET_OS_MAC
 CFLAGS += -DWINOBJCRT_IMPEXP=
+CFLAGS += -DLOGGING_IMPEXP=
 CFLAGS += -mmacosx-version-min=$(OSX_VERSION_MIN)
 
 LDFLAGS := -framework Foundation -mmacosx-version-min=$(OSX_VERSION_MIN)

